docs/02-basic-tutorial.md

```diff
@@ -77,8 +77,6 @@ Pages (route handlers) in Reflex are React components. They all share the same `
 * `app-state` - the application state (more on that later)
 * `context` - the parsed URL with parameters (route segments or query string)
 
-To learn more about routing, read the [routing guide]().
```

Comment:
Actually this already exists, this should link to `08-isomorphism-routing.md`

But the text above is wrong. With a recent change the props ara no longer `app-state` and `context`, it's just `app-state`, which has two top level keys - `state` and `route`. Needs a larger fix :(